packaging/build_ffmpeg.sh

```diff
@@ -78,27 +79,31 @@ if [[ "$(uname)" == Darwin ]]; then
         avdevice=libavdevice.58
         avfilter=libavfilter.7
         swscale=libswscale.5
+        swresample=libswresample.3
     elif [[ ${major_ver} == 5 ]]; then
         avutil=libavutil.57
         avcodec=libavcodec.59
         avformat=libavformat.59
         avdevice=libavdevice.59
         avfilter=libavfilter.8
         swscale=libswscale.6
+        swresample=libswresample.4
     elif [[ ${major_ver} == 6 ]]; then
         avutil=libavutil.58
         avcodec=libavcodec.60
         avformat=libavformat.60
         avdevice=libavdevice.60
         avfilter=libavfilter.9
         swscale=libswscale.7
+        swresample=libswresample.4
```

Comment:
Yup, it's version 4 twice in a row, this isn't a typo (logs were previously failing)!